chain/default_syncer.go

```diff
@@ -208,6 +210,31 @@ func (syncer *DefaultSyncer) syncOne(ctx context.Context, parent, next types.Tip
 		return nil
 	}
 
+	pMinTs, err := parent.MinTimestamp()
+	if err != nil {
+		return err
+	}
+
+	tickets, err := next.Tickets()
+	if err != nil {
+		return err
+	}
+
+	for _, b := range next.ToSlice() {
+		if uint64(b.Timestamp) > uint64(time.Now().UTC().Unix()) {
+			return errors.Errorf("block: %s was generated by miner: %s too far in the future", b.Cid(), b.Miner.String())
+		}
+		limit := int64(pMinTs) + (int64(syncer.blockTime.Seconds()) * int64(len(tickets)))
+		// By setting this to strictly less than, we allow for a 1 second drift
+		// in clocks. This has an added benefit in that it keeps FAST tests
+		// working that mine at the sub-second level.
+		// The loss of precision in seconds need not matter.
+		if int64(b.Timestamp) < limit {
```

Comment:
Please submit a PR to the spec with this change (can happen after you merge this one)

Reply:
spec change being: `b.Timestamp <= limit` to strictly ` b.Timestamp < limit`?